components/hab/src/command/plan.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod initialize {
+    use std::fs::create_dir_all;
+    use std::fs::File;
+    use std::io::Write;
+    use std::path::Path;
+    use std::collections::HashMap;
+
+    use handlebars::Handlebars;
+
+    use common::ui::{UI, Status};
+    use error::Result;
+
+    const PLAN_TEMPLATE: &'static str = "plan {{origin}}/{{name}}";
+    const RUN_HOOK_TEMPLATE: &'static str = "hook {{origin}}/{{name}}";
+
+    pub fn start(ui: &mut UI, origin: String, name: String) -> Result<()> {
+        try!(ui.begin("Constructing a cozy habitat for your app..."));
+        try!(ui.br());
+
+        // Build out the variables passed.
+        let handlebars = Handlebars::new();
+        let mut data = HashMap::new();
+        data.insert("name".to_string(), name);
+        data.insert("origin".to_string(), origin);
+
+        let rendered_plan = try!(handlebars.template_render(PLAN_TEMPLATE, &data));
```

Comment:
After reviewing hooks it's better not to since we don't want ot run handlebars on the hooks initially.